client/src/components/ui/itemrow/util.ts

```diff
@@ -0,0 +1,5 @@
+import { Document } from '~model/document';
+
+export function redirectHandler(id: Document['id']) {
+    window.location.href = `/track?id=${id}`;
+}
```

Comment:
One last suggestion! 🚀
```suggestion
export function goToTrackingPage(id: Document['id']) {
    window.location.href = `/track?id=${id}`;
}
```